worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/WorkflowManager.java

```diff
@@ -78,6 +90,14 @@ public Workflow get(final String workflowName){
 
                 final StringBuilder stringBuilder = new StringBuilder();
                 stringBuilder.append(String.format("var ACTIONS = %s;\n", gson.toJson(workflow.getActions())));
+
+                if (contextScriptFile != null && contextScriptFile.exists()) {
+                    final String contextScriptFileContent = FileUtils.readFileToString(contextScriptFile, StandardCharsets.UTF_8);
+                    stringBuilder.append(contextScriptFileContent);
+                }else{
+                    LOG.warn("Context script file from the path " + contextScriptFilePath + " not found.");
```

Comment:
This is still wrong.  The way you have the code now it will warn even if `contextScriptFile` was not configured.  This was my original comment for your reference:
> Log a warning if the script file has been configured but is not found.

Reply:
Updated to log warning only if the file not exists.